ops/terraform/services/pipeline/user-data.sh.tftpl

```diff
@@ -28,9 +35,9 @@ aws ssm get-parameters-by-path \
 cat <<EOF > extra_vars.json
 {
   "data_pipeline_db_url": "${writer_endpoint}",
-  "data_pipeline_jvm_args": "-Xmx{{ ansible_memtotal_mb * 0.80 | int }}m -XX:+PreserveFramePointer",
+  "data_pipeline_jvm_args": "-Xms{{ ((ansible_memtotal_mb * 0.80) | int) }}m -Xmx{{ ((ansible_memtotal_mb * 0.80) | int) }}m -XX:+PreserveFramePointer",
   "data_pipeline_loader_threads": "{{ ansible_processor_vcpus * 25 }}",
-  "data_pipeline_new_relic_app_name": "BFD Pipeline ({{ env_name_std }})",
+  "data_pipeline_new_relic_app_name": "BFD Pipeline ${pipeline_instance} ({{ env_name_std }})",
```

Comment:
Shoot. Sorry to waffle on this decisions, but based on the most recent comments from @brianburton, maybe we want a tiny bit more sophistication here. 

**If you get to it**, templating a conditional to stick the ${pipeline_instance} on the application name for non-rda pipelines seems like the best course of action now. We don't use NR extensively for anything ccw-related, so we may as well disrupt those rda workload visualizations as little as possible. **Otherwise**, I could submit a change to this branch if you'd entertain that again?

Reply:
I'm sorry I didn't see this until today - go for it! And thank you.